src/bo4e/com/zeitspanne.py

```diff
@@ -0,0 +1,18 @@
+"""
+contains the COM Zeitspanne
+"""
+from datetime import datetime
+from typing import Optional
+
+from bo4e.com.com import COM
+
+
+class Zeitspanne(COM):
```

Comment:
Über den Begriff müssten wir vielleicht doch nochmal genauer nachdenken, weil es wohl doch eher synonym zu Zeitintervall ist. ich mach mal ein issue auf, aber merge das hier jetzt trotzdem

Reply:
Ah moment: das ist hier noch falsch, Zeitintervall und Zeitraum müssen noch gelöscht werden

Reply:
Zeitintervall hab ich gelöscht, aber Zeitraum wird ne größere Sache, das müssen wir in einem anderen PR angehen